pkg/timer/runtime/worker.go

```diff
@@ -395,7 +395,7 @@ func (w *hookWorker) triggerEvent(hook api.Hook, req *triggerEventRequest, logge
 
 		if err != nil {
 			w.onSchedEventErrCounter.Inc()
-			logger.Error(
+			logger.Warn(
 				"error occurs when invoking hook OnTimerEvent",
```

Comment:
When this call to OnTimerEvent causes an error, is it necessary to report an error message?

Reply:
It can try again, which means it is not a serious problem

Reply:
Got it.